### ISSUE TITLE
Mesh goes black sometimes

### ISSUE THREAD

**Issue by MattJeanes** [MattJeanes/Safe-Space#4]
This can either be partial or complete. Not sure what causes it yet but it's only on spawn, respawning it usually fixes it or moves around the black areas.

**Comment by LoganDark** [comment]
Always black. Portal doesn't work.

**Comment by LoganDark** [comment]
Nevermind.. the portal does work. However, it's still always black for me.

**Comment by Cool-Dude-Fab** [comment]
The safe space is always staying black no matter what i do. I have tried to respawn it constantly but it never works (i once did this for like 10 min straight). Is it a material i dont have or something or is it because of the setting or map? I tried in flatgrass and it didnt work, i didnt change the settings at all. Maybe its because im on a mac but its strange because usually for addons that work on a pc its the addon itself that doesnt work instead of the materials not working. Safe space works completeley fine (like i can go in and out with easy and so can props) but its a shame that i cant see anything inside and its so dark and i also cant see the corners so i dont know where the limit is. Please help me or if there is a solution. Btw safe space is like one of my fav addons and would be a shame if it didnt work properly

**Comment by LoganDark** [comment]
It's because you have a Mac. I have the same issue. Use Boot Camp or there
are a few materials that don't go black.

**Comment by Cool-Dude-Fab** [comment]
im not gonna use bootcamp (cuz doesnt that cost money or something) and could u tell me which materials may work?

**Comment by LoganDark** [comment]
I don't remember which materials specifically, but Boot Camp does not cost
money. You can use Windows without activating it.

**Comment by Cool-Dude-Fab** [comment]
i will try a few material §(if not all) but for bootcamp it says u need a A Microsoft Windows installation media or disk image (ISO) containing a 64-bit version of Microsoft Windows 8 or later

**Comment by LoganDark** [comment]
Yes, contrary to popular belief, you need to download Windows to install
Windows, you can download it for free on Microsoft's website

**Comment by Cool-Dude-Fab** [comment]
im doin that right now but can u switch between mac and windows? If so, is it easy?

**Comment by LoganDark** [comment]
yeah, all you have to do is reboot and hold down option at the chime

**Comment by Cool-Dude-Fab** [comment]
what the hell does that mean? how do u reboot? what is the option key?

**Comment by LoganDark** [comment]
You know like, the 'ding' sound when your Mac starts up, when you hear
that, press Option and hold it down
